src/storage/ipfs-block.js

```diff
@@ -38,14 +38,11 @@ const IPFSBlockStorage = async ({ ipfs, timeout, pin } = {}) => {
    */
   const put = async (hash, data) => {
     const cid = CID.parse(hash, base58btc)
-    await ipfs.block.put(data, {
-      cid: cid.bytes,
-      version: cid.version,
-      format: 'dag-cbor',
-      mhtype: 'sha2-256',
-      pin,
-      timeout
-    })
+    await ipfs.blockstore.put(cid, data)
```

Comment:
Do we have a way to control the timeout time anymore? If not, we could remove the `timeout` option from the `IPFSBlockStorage` function above?

Reply:
I'm not seeing it documented anywhere so probably no longer applies.

Can probably remove:

https://github.com/orbitdb/orbitdb/blob/0a5f8b54e3dac37ebf0ef9446809323588ac75d6/src/storage/ipfs-block.js#L10

and 

https://github.com/orbitdb/orbitdb/blob/0a5f8b54e3dac37ebf0ef9446809323588ac75d6/src/storage/ipfs-block.js#L29

and remove the timeout param from https://github.com/orbitdb/orbitdb/blob/0a5f8b54e3dac37ebf0ef9446809323588ac75d6/src/storage/ipfs-block.js#L26C41-L26C48.